go/apps/api/routes/v2_keys_remove_roles/handler.go

```
@@ -218,7 +218,9 @@ func (h *Handler) Handle(ctx context.Context, s *zen.Session) error {
 		}
 
 		rolePermissions := make([]db.Permission, 0)
-		json.Unmarshal(role.Permissions.([]byte), &rolePermissions)
+		if permBytes, ok := role.Permissions.([]byte); ok && permBytes != nil {
+			_ = json.Unmarshal(permBytes, &rolePermissions) // Ignore error, default to empty array
```

golangci-lint (2.2.2)

the given struct should be annotated with the `json` tag

(musttag)